### DISCUSSION TITLE
Welcome to Awesome-Dialogue-State-Tracking Discussions!

### DISCUSSION THREAD

**Discussion by yukyunglee** [yukyunglee/Awesome-Dialogue-State-Tracking#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by squiduu** [comment]
Hi, my name is Taejin Woo.

I am studying DST at PRML in Korea University.

I visit this repository almost everyday, and I always get very useful information from you, thank you.

So, I have several DST-related papers that can be added to this repository if you do not feel uncomfortable.


 1. Effective Sequence-to-Sequence Dialogue State Tracking (Pegasus) - https://arxiv.org/pdf/2108.13990.pdf
 2. Improving Dialogue State Tracking with Turn-based Loss Function and Sequential Data Augmentation - https://aclanthology.org/2021.findings-emnlp.144.pdf
 3. Multi-Task Pre-Training for Plug-and-Play Task-Oriented Dialogue System (PPTOD) - https://arxiv.org/pdf/2109.14739.pdf
 4. SCoRe: Pre-Training for Context Representation in Conversational Semantic Parsing - https://openreview.net/pdf?id=oyZxhRI2RiE
 5. Amendable Generation for Dialogue State Tracking (AG-DST) - https://aclanthology.org/2021.nlp4convai-1.8.pdf
 6. Improving Longer-range Dialogue State Tracking - https://arxiv.org/pdf/2103.00109.pdf
 7. Comprehensive Study: How the Context Information of Different Granularity Affects Dialogue State Tracking?` - https://arxiv.org/pdf/2105.03571.pdf
 8. Parallel Interactive Networks for Multi-Domain Dialogue State Generation (PIN) - https://aclanthology.org/2020.emnlp-main.151.pdf
 9. DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue - https://arxiv.org/pdf/2009.13570.pdf

Thank you for your consideration.

**Reply by yukyunglee** [reply]
안녕하세요 태진님 🤗,

논문 리스트 공유해주셔서 감사드립니다.
말씀해주신 논문 리스트중에서 이미 해당 레포에 포함된 논문들이 있네요
중복되는 논문들은 제외하고 추가하도록 하겠습니다.
앞으로는 pull request로 논문 추가해주셔도 좋습니다 :) 

감사합니다

**Reply by squiduu** [reply]
제가 미처 리스트를 다 확인 못 했네요
혹시 또 추가할 수 있는 논문이 있으면 PR 드리겠습니다

감사합니다
